hadoop-hdds/erasurecode/src/main/java/org/apache/ozone/erasurecode/rawcoder/ByteBufferEncodingState.java

```diff
@@ -32,6 +34,8 @@
   ByteBuffer[] inputs;
   ByteBuffer[] outputs;
   boolean usingDirectBuffer;
+  public static final Logger LOG =
+      LoggerFactory.getLogger(ByteBufferEncodingState.class);
```

Comment:
I guess we don't need the Logger or imports now if new log message is removed now?